README.md

```diff
@@ -36,6 +38,9 @@ Convert the following ActiveRecord sequence to Mongoose:
 
 ```js
 // Your answer...
+
+
+
```

Comment:
check out the solution - https://github.com/ga-wdi-quizzes/week-10/tree/solution